nextcord/utils.py

```diff
@@ -497,6 +500,17 @@ def _bytes_to_base64_data(data: bytes) -> str:
     return fmt.format(mime=mime, data=b64)
 
 
+async def _obj_to_base64_data(obj: Optional[Union[bytes, Attachment, Asset, File]]) -> Optional[str]:
```

Comment:
Fix return type, may be typehinted as `Optional[Union[str, None]]` instead

Reply:
The way it is seems more correct to me.

`Optional[str]` means it returns a string or None (it means the same as `Union[str, None]` but more concise)

Reply:
> Fix return type, may be typehinted as `Optional[Union[str, None]]` instead

`Optional[T]` already means `Optional[Union[T, None]]`

Reply:
Holy foof yes, I forgor typehinting, thanks